test/functional/ui/popupmenu_spec.lua

```diff
@@ -6673,7 +6685,7 @@ describe('builtin popupmenu', function()
       feed('<C-E>')
     end)
 
-    it('supports mousemodel=popup', function()
+    it('testme supports mousemodel=popup', function()
```

Comment:
:eyes:

Reply:
Oops, that must have remained, when I debugged some issues yesterday or the day before

Reply:
fixed